src/main/java/io/ddavison/conductor/Locomotive.java

```diff
@@ -495,15 +495,18 @@ public Locomotive scrollTo(By by) {
     }
 
     /**
-     * Scroll to a specified element
+     * Scroll to a specified element and attempt to center it in the viewport
+     *
+     * See <a href="https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView">MDN web docs Element.scrollIntoView()</a>
      *
      * @param element to scroll to
      * @return This instance for method chaining.
      */
     public Locomotive scrollTo(WebElement element) {
         // Execute javascript to scroll to the element.
-        ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView()", element);
-
+        ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView({" +
+                "block: \"center\"" +
```

Comment:
I think scrolling to the center is a good default.

In the future it would probably be good to provide an additional method signature with an enum to choose to which part we want to scroll:
```
public Locomotive scrollTo(WebElement element, ElementTarget target);
enum ElementTarget { TOP, CENTER, BOTTOM };
```
or similar.

But as always YAGNI. Let's not do it until we need it.